ui/framework/src/ui-framework/zones/toolsettings/ToolSettingsManager.ts

```diff
@@ -141,4 +141,19 @@ export class ToolSettingsManager {
     return ToolSettingsManager._toolIdForToolSettings;
   }
 
+  public static focusIntoToolSettings(): boolean {

```

Comment:
It appears that this will work in 1.0 and when docked in 2.0 but not when ToolSettings is in floating state. We need to make sure the floating state version is wrapped with a div you can locate.

Reply:
I added "uifw-tool-settings-grid-container" to ToolSettingsGridContainer, but that might not be enough.

Reply:
I completely missed that, sorry. I was just looking at my local build.